entity-service/src/main/kotlin/com/egm/stellio/entity/repository/Neo4jRepository.kt

```diff
@@ -547,17 +546,18 @@ class Neo4jRepository(
             .first()
     }
 
-    fun filterExistingEntitiesAsIds(entitiesIds: List<String>): List<String> {
+    fun filterExistingEntitiesAsIds(entitiesIds: List<URI>): List<URI> {
         if (entitiesIds.isEmpty()) {
             return emptyList()
         }
 
         val query = "MATCH (entity:Entity) WHERE entity.id IN \$entitiesIds RETURN entity.id as id"
 
-        return session.query(query, mapOf("entitiesIds" to entitiesIds), true).map { it["id"] as String }
+        return session.query(query, mapOf("entitiesIds" to entitiesIds.map { it.toString() }), true)
```

Comment:
I see a lot of `entitiesIds.map { it.toString() }`

You could have added a `List<URI>.toString()`, thoughts?

Reply:
i'll add it